docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -963,6 +1005,42 @@ function _layout_builder_custom_process_element(array $element, FormStateInterfa
         break;
 
       case 'inline_block:uiowa_card':
+        $style_options = [
```

Comment:
Can we call this `$media_types`? I think style is confusing in this context (since we have LB styles too).